include/openssl/opensslv.h

```diff
@@ -40,11 +40,7 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x10101000L
-# ifdef OPENSSL_FIPS
-#  define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1-fips-dev  xx XXX xxxx"
-# else
-#  define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1-dev  xx XXX xxxx"
-# endif
+# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1-dev  xx XXX xxxx"
 
```

Comment:
I'm not so sure it will be a problem.  I just looked through the scripts, and it looked like it only fixes the dates if the lines in question are present.  With the fips line gone, they will probably simply ignore that.

It was just a glance, though, so I may have missed some subtleties...